package.json

```diff
@@ -18,11 +18,11 @@
   "author": "mesur.io <oss@mesur.io>",
   "license": "MIT",
   "dependencies": {
-    "@capacitor/core": "^3.0.0"
+    "@capacitor/core": "^4.0.1"
```

Comment:
For thinking the future, I think Capacitor versions should not be fixed except for major versions.